backend/Events/GameStateEvents/NightStateChangeEvents.js

```diff
@@ -47,9 +45,14 @@ function endNight(io, socket, mafiaGame) {
 
     const winningRole = room.getWinningRole();
     if (winningRole !== null) {
-        io.in(roomID).emit('game-over', new GameOverDTO(winningRole, room.getPlayersByRole(winningRole)));
+        io.in(roomID).emit(
+            'game-over',
+            new GameOverDTO(
+                winningRole,
+                room.getPlayersByRole(winningRole).map((p) => p.nickname)
```

Comment:
did we fix this one yet? where winning role said civilian and it didnt incie detective or medic

Reply:
oh we fixed it and it'll be included in the next PR!